runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -351,6 +364,50 @@ public void onVarbitChanged(VarbitChanged event)
 		}
 	}
 
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() == WidgetID.FISHING_TRAWLER_GROUP_ID)
+		{
+			startTime = Instant.now();
+		}
+	}
+
+	/**
+	 * Changes the Fishing Trawler timer widget from minutes to minutes and seconds
+	 */
+	private void updateTrawlerTimer()
+	{
+		int regionID = client.getLocalPlayer().getWorldLocation().getRegionID();
+
+		if (regionID != TRAWLER_SHIP_REGION_NORMAL && regionID != TRAWLER_SHIP_REGION_SINKING || startTime == null)
+		{
+			return;
+		}
+
+		Widget trawlerTimerWidget = client.getWidget(WidgetID.FISHING_TRAWLER_GROUP_ID, 37);
```

Comment:
This probably should have a `WidgetInfo` created for whatever this is. Also you can probably just check for the widget existing to tell if they are in the trawler or not.